packages/camera/camera/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.9.4+5
```

Comment:
None of these need version bumps; you can just use `NEXT`. Test-only/example-only changes are exempt from version changes because they don't affect clients of the packages.

Reply:
Done

Reply:
After all, I fixed a bug here. I'm bumping the version, then